homeassistant/components/unifiprotect/sensor.py

```diff
@@ -399,6 +401,59 @@ def _get_alarm_sound(obj: Sensor) -> str:
 )
 
 
+CAMERA_TRIP_SENSORS: tuple[ProtectSensorEntityDescription, ...] = (
+    ProtectSensorEntityDescription(
+        key="doorbell_last_trip_time",
+        name="Last Doorbell Ring",
+        device_class=SensorDeviceClass.TIMESTAMP,
+        icon="mdi:doorbell-video",
+        ufp_required_field="feature_flags.has_chime",
+        ufp_last_trip_value="last_ring",
+    ),
+)
+
+LIGHT_TRIP_SENSORS: tuple[ProtectSensorEntityDescription, ...] = (
+    ProtectSensorEntityDescription(
+        key="motion_last_trip_time",
+        name="Last Motion Detected",
+        device_class=SensorDeviceClass.TIMESTAMP,
+        ufp_last_trip_value="last_motion",
+    ),
+)
+
+SENSE_TRIP_SENSORS: tuple[ProtectSensorEntityDescription, ...] = (
+    ProtectSensorEntityDescription(
+        key="door_last_trip_time",
+        name="Last Contact",
```

Comment:
```suggestion
        name="Last Open",
```

? It looks like they are all "opening" devices window, door, garage door. etc.

Reply:
4d8a346a58e03697d5ccb57c52f1b5c3055c1f49